fastlane/Screengrabfile

```diff
@@ -0,0 +1,15 @@
+# remove the leading '#' to uncomment lines
+
+app_package_name 'org.flyve.inventory.agent'
+use_tests_in_packages ['org.flyve.inventory.agent']
+
+app_apk_path 'app/build/outputs/apk/app-debug.apk'
+tests_apk_path 'app/build/outputs/apk/app-debug-androidTest.apk'
+
+locales ['en-US', 'en-GB', 'fr-FR', 'es-ES', 'ko-KR', 'ru-RU']
```

Comment:
If we download a new version of the Transifex, a new locale is pushed to this array?

Reply:
No, need to be manually

Reply:
ToDo